neuro_extras/main.py

```diff
@@ -257,28 +262,44 @@ async def _data_cp(
         raise ValueError(
             "This command can't be used to copy data between two storage locations"
         )
+    if source_url_type == UrlType.DISK and destination_url_type == UrlType.DISK:
+        raise ValueError(
+            "This command can't be used to copy data between two persistent disks"
+        )
```

Comment:
also: disk<->storage. Better implement more generic logic:
1) if source_url_type == destination_url_type: raise not supported
2) else if sorted(source_url_type, destination_url_type) == sorted(UrlType.DISK, UrlType.STORAGE): raise not supported
3) else if any(url in (UrlType.STORAGE, UrlType.DISK) for url in (source_url_type, destination_url_type)): spawn a job
4) else if any(url == UrlType.LOCAL for url in (source_url_type, destination_url_type)): just copy
5) else: raise ValueError

Reply:
> also: disk<->storage

Why not? The code supports this option and I can see a legitimate use-case.

> if source_url_type == destination_url_type: raise not supported

This conflicts with local ⇔ local